wbc_pr2_ctrl_pump_plugin:
  type: PumpPlugin
  com_enabled: y
